ols/utils/config.py

```diff
@@ -56,4 +56,4 @@ def init_config(config_file: str) -> None:
     except Exception as e:
         print(f"Failed to load config file {config_file}: {e!s}")
         print(traceback.format_exc())
-        raise e
+        raise
```

Comment:
it may have been redundant, but what was the harm?  I like that it was explicit about what exception is being raised w/o having to know the nuances of python's implicit behaviors.  I wouldn't have made someone add the arg in if they didn't have it in the original PR, but did we really need to remove it?